gce_image_import_export_tests/main.go

```diff
@@ -16,23 +16,22 @@ package main
 
 import (
 	"context"
-	"github.com/GoogleCloudPlatform/compute-image-tools/go/e2e_test_utils"
 	"log"
 	"regexp"
 	"sync"
 
 	"github.com/GoogleCloudPlatform/compute-image-tools/gce_image_import_export_tests/test_suites/export"
 	"github.com/GoogleCloudPlatform/compute-image-tools/gce_image_import_export_tests/test_suites/import"
+	"github.com/GoogleCloudPlatform/compute-image-tools/go/e2e_test_utils"
 	"github.com/GoogleCloudPlatform/compute-image-tools/go/e2e_test_utils/junitxml"
 	"github.com/GoogleCloudPlatform/compute-image-tools/go/e2e_test_utils/test_config"
 )
 
-var testFunctions = []func(context.Context, *sync.WaitGroup, chan *junitxml.TestSuite, *log.Logger,
-	*regexp.Regexp, *regexp.Regexp, *testconfig.Project){
-	importtestsuites.TestSuite,
-	exporttestsuites.TestSuite,
-}
-
 func main() {
-	e2etestutils.LaunchTests(testFunctions, "[ImageImportExportTests]")
+	e2etestutils.LaunchTests([]func(context.Context, *sync.WaitGroup, chan *junitxml.TestSuite, *log.Logger,
```

Comment:
originally we run export and import suites in parallel. Now I changed it to sequencial, because they will conflict due to updating gcloud version.